### PR TITLE
[WIP] Remove `eth-lib` package and replace with `ethereumjs-utils` functions

### LINKED ISSUE #4095: Remove eth-lib
[eth-lib](https://github.com/maiavictor/eth-lib#readme) has not been maintained or quite sometime now, and I believe that the ethereumjs [monorepo](https://github.com/ethereumjs/ethereumjs-monorepo) should contain everything we need.

### PR DESCRIPTION
**Pull request by akshatamohanty**
## Description
Fixes #4095 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

**Comment by GregTheGreek** [comment]
Hey @akshatamohanty thanks for the contirbution, just a heads up you should be targeting `ethereumjs/util` not `ethereumjs-util`

**Comment by akshatamohanty** [comment]
@GregTheGreek using [this](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util) from the `ethereumjs/ethereumjs-monorepo`.  The README seems to point back to `npm install ethereumjs-util`. What am I missing?

**Comment by GregTheGreek** [comment]
Sorry you're correct, looks like they haven't cut a new release yet

**Comment by GregTheGreek** [comment]
So my pr #4083 actually could benefit from this PR. The final thing is tackling the `Account` package (which is what I need). LMK if you need help

File: packages/web3-eth-accounts/src/index.js [hunk 1]
```diff
@@ -42,6 +39,49 @@ var isNot = function(value) {
     return (_.isUndefined(value) || _.isNull(value));
 };
 
+var flattenBytes = function(a) {
```
Comment (akshatamohanty) [inline packages/web3-eth-accounts/src/index.js:42]: @GregTheGreek Yes, I'm getting stuck with the `Account` package. There don't seem to be direct mappings in `ethereumjs-utils` for most of it. I've added the functions from `eth-lib` inline, within the file as helpers (has missing references right now) and planning to cleanup each function to use internal utils instead. If they don't exist already, will add them to `web3-utils` (eg, Bytes.flatten, Bytes.slice etc).  Does that sound like the right approach?

Comment (GregTheGreek) [reply packages/web3-eth-accounts/src/index.js:42]: I think a lot of the functions you want might be [here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/util/src/signature.ts) to remove code in `hashMessage`?

Also for the `recoverTransaction` you're going to run into some problems becasue of berlin support, maybe @luu-alex can make a gist for you showing how it should be supported?

**Comment by GregTheGreek** [comment]
We covered this in the berlin hardfork

**Pull request closed without merging**